cypress/support/awx-commands.ts

```diff
@@ -709,40 +709,51 @@ Cypress.Commands.add(
   }
 );
 
-Cypress.Commands.add('createAwxConstructedInventory', (organization: Organization) => {
-  const arrayOfInventories: number[] = [];
-  // creates 3 inventories
-  for (let i = 0; i < 3; i++) {
-    cy.createAwxInventory(organization).then((inv: Inventory) => {
-      arrayOfInventories.push(inv.id);
-    });
-  }
-  cy.requestPost<Partial<Inventory & { inventories: Array<number | undefined> }>>(
-    awxAPI`/constructed_inventories/`,
-    {
+Cypress.Commands.add(
+  'createAwxConstructedInventory',
+  (organization: Organization, source_vars?: boolean) => {
+    const arrayOfInventories: number[] = [];
+    // creates 3 inventories
```

Comment:
should this be part of the command? this will always create 3 inventories

Reply:
The original command had this creation too. Im leaving it as optional and adding params with number of created inventories.